### PR TITLE
Release 24

### PR DESCRIPTION
**Pull request by amglebihan**
Test URLs:
- Before: https://main--vg-macktrucks-com--hlxsites.hlx.page/
- After: https://release-24--vg-macktrucks-com--hlxsites.hlx.page/

### CODE REVIEW

**Review by jonatan-lledo-netcentric — COMMENTED** [review]
the changes in sub-nav.js are "_on-testing_" status, so we can leave them or remove them until the tests are finished

**Review by jonatan-lledo-netcentric — APPROVED** [review]

File: scripts/scripts.js [hunk 1]
```diff
@@ -415,7 +417,7 @@ const createInpageNavigation = (main) => {
     subnavLink.textContent = item.title;
 
     subnavItem.append(subnavLink);
-    navItems.push(subnavItem);
+    navItems.push(subnavLink);
```
Comment (cincplug) [inline scripts/scripts.js:420]: **subNav** would be more readable than **subnav** and it's subNav already elsewhere in the code, so let's better have it that way

Comment (amglebihan) [reply scripts/scripts.js:420]: Only the parameter was changed there and merged to develop, we usually don't do changes directly on the release branch, but I can do it if you think it's worth it.

Comment (cincplug) [reply scripts/scripts.js:420]: Aha well sure, if it's already in release then let's do it some other time :)

**Review by santi-homps — APPROVED** [review]

**Pull request merged**